packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -7772,6 +7773,46 @@ QUnit.module('label integration', () => {
 });
 
 QUnit.module('accessibility', () => {
+    QUnit.test('root element should have arialabelledby attribute based from multitag id', function(assert) {
+        const sampleData = ['First Item', 'Second Item', 'Third Item', 'Fourth Item'];
+        const $tagBox = $('#tagBox').dxTagBox({
+            items: sampleData,
+            value: sampleData,
+            maxDisplayedTags: 2,
+            showMultiTagOnly: false,
+        });
+        const $multiTagContainer = $tagBox.find(`.${TAGBOX_MULTI_TAG_CLASS}`);
+        const $parentTagContainerIds = $tagBox.attr('aria-labelledby').split(' ');
+
+
+        assert.strictEqual($parentTagContainerIds.length, 2, 'root element aria-labelledby attribute contains two ids');
+        assert.strictEqual($parentTagContainerIds.includes($multiTagContainer.attr('id')), true, 'aria-labelledby attribute contains multitag id');
+    });
+
+    QUnit.test('check showmultitag option is showing the right count of tags correctly in true and false scenarios', function(assert) {
+        [true, false].forEach(showMultiTagOnly => {
```

Comment:
Let's split this configuration into separate tests

Reply:
Ok noted

Reply:
[6a3a6dc](https://github.com/DevExpress/DevExtreme/pull/27063/commits/6a3a6dc948070b72983e42ce34c7c7d9f1c6b1aa)